styles.css

```diff
@@ -527,3 +527,102 @@ body {
   bottom: 74.49%;
   opacity: 0.5;
 }
+
+.contact-section {
+  position: absolute;
+  width: 100%;
+  height: 813px;
+  left: 0;
+  top: 4288px;
+  background: #f9f9fa;
+}
+
+.contact-form {
+  padding: 0 20px;
+}
+
```

Comment:
- [ ] Your webpage is overflowing. Kindly fix this by giving if an `overflow-x: hidden;` 🙏 

![20220705-225524_capture](https://user-images.githubusercontent.com/74814780/177424634-5a42875f-9677-4ecf-af2c-fb7629c44bb9.gif)

Reply:
Fixed